inspirehep/modules/records/wrappers.py

```diff
@@ -68,6 +69,46 @@ def get_link_info_for_external_sys_identifiers(self, extid, ext_sys_id_info):
             return {'url_name': ext_sys_id_info[0],
                     'url_link': ext_sys_id_info[1] + extid}
 
+    @staticmethod
+    def get_enhanced_author(author):
+        """Enhances the author object with name fields that are needed for the UI."""
+        item = copy(author)
+        names = author['full_name'].split(',', 1)
+        if len(names) > 1:
+            item.update({
+                'first_name': names[1].replace(',', '').strip(),
```

Comment:
@drjova check `replace` for dealing qith `Jr.` case. There is also a test for it!